docs/docs/Components/components-data.mdx

```diff
@@ -187,6 +187,64 @@ The output of the **File** component depends on the number and type of files loa
 
 - **Multiple files**: Produces a **Files** [`DataFrame`](/data-types#dataframe) containing the content and metadata of all selected files.
 
+## Create Knowledge
+
+The **Create Knowledge** component creates or appends to a [Langflow Knowledge Base](/knowledge) from a DataFrame.
+
+To create a knowledge base in a flow, see the **Create Knowledge** template.
+
+For more information, see the [Knowledge Base documentation](/knowledge).
+
+### Create Knowledge parameters
+
+Most **Create Knowledge** component input parameters are hidden by default in the visual editor.
```

Comment:
Please modify line 200 depending on the truth of that statement.
I sometimes change this depending on how many are hidden (example: _Some parameters are hidden_ vs _Most_).

If none are hidden, you can omit line 200 and 201.